tools/collect_logs/collect_current_logs.sh

```diff
@@ -18,16 +18,15 @@ tarball="log/evm_current_$(uname -n)_$(date +%Y%m%d_%H%M%S).tar.xz"
 if [[ -n "$APPLIANCE_PG_DATA" && -d "$APPLIANCE_PG_DATA/pg_log" ]]; then
     echo "This ManageIQ appliance has a Database server and is running version: $(psql --version)"
     echo " Log collection starting:"
-    XZ_OPT=-9 tar -cJvf ${tarball} --sparse -X $collect_logs_directory/exclude_files   BUILD GUID VERSION REGION log/*.log log/*.txt config/* /var/log/* log/apache/* $APPLIANCE_PG_DATA/pg_log/* $APPLIANCE_PG_DATA/postgresql.conf
+    XZ_OPT=-9 tar -cJvf ${tarball} --sparse --warning=no-file-changed -X $collect_logs_directory/exclude_files --exclude="/var/log/*.log*" --exclude="/var/log/messages-*" --exclude="/var/log/journal/*/system@*.journal" BUILD GUID VERSION log/*.log log/*.txt config/* /var/log/* log/apache/*.log $APPLIANCE_PG_DATA/pg_log/* $APPLIANCE_PG_DATA/postgresql.conf
```

Comment:
`--exclude="/var/log/messages-*"` Technically there are a few other log files that look like `/var/log/file-timestamp` but without a regex I couldn't distinguish between this and e.g. `/var/log/qemu-ga/`

Reply:
I have a dumb question.  Is there any value in maybe doing it by mtime or ctime?  I purposely didn't read your comment above until after I read the change and I had similar thoughts about possibly missing something.

Reply:
You mean e.g. only include logs which were modified during the current day?  That is tricky because the log files aren't rotates daily, usually they are only rotated once they get over a certain size.  Also we might want to include "current" logs that aren't logged to constantly like `/var/log/dnf.log` if they installed packages 3 days ago and collect logs today it would be excluded.

Reply:
yeah, I was trying to avoid trying to deal with the different naming conventions.

Reply:
If we just want a "quick win" for backport, the bulk of the size is coming from rolled-over `/var/log/messages-*` and `/var/log/journal/*/systemd@*.journal` so we could ignore the "general" rollover log naming conventions and fix just these two.

Reply:
Maybe we can do that and ask some of the users who reported this problem if they can verify it picks up the wrong or right files for them?